src/components/Directory/DirectoryItem.js

```diff
@@ -12,6 +13,13 @@ function DirectoryItem(props) {
             ? icons[props.user.subType]
             : icons[props.user.type]}
           {props.user.name}
+          {props.user.onionUrl !== undefined && (
+            <a href={props.user.onionUrl} className="onionIcon" rel="noopener noreferrer">
+              <svg viewBox="0 0 1000 500">
+                <use href={iconSprite + "#onion"} />
+              </svg>
+            </a>
+          )}
```

Comment:
This link is nested within the other link, can you correct that?

Reply:
okay, will check tmrw

Reply:
> This link is nested within the other link, can you correct that?

Removed nested link